### PR TITLE
Let's add OnlyOffice

### PR DESCRIPTION
**Pull request by jeditobe**
OnlyOffice (formerly TeamLab), a powerful and free software office suite developed by Ascensio System SIA. Supports opening and saving documents in Microsoft Office formats.

It works in ReactOS quite nice and stable! Tested by me and some other people.

https://twitter.com/AgeNuclear/status/1628793519388205056

### CODE REVIEW

**Review by JoachimHenze — CHANGES_REQUESTED** [review]
Please correct your previous PR first (the one with adding GPU-Z with the inaccessible URL before merging the next thing into rapps-DB)

**Comment by learn-more** [comment]
> Please correct your previous PR first (the one with adding GPU-Z with the inaccessible URL before merging the next thing into rapps-DB)

That is not what reviews are for, a review is to be about this PR, not about a previous one.

**Comment by jeditobe** [comment]
[JoachimHenze](https://github.com/JoachimHenze) I see no problems with #193, I think it should be approved

**Review by JoachimHenze — APPROVED** [review]
My concern has been addressed.
Sure this didn't invalidate the quality of PR193. But it is the kind of responsibility that we can/should expect from our contributors. As each app added into the DB will induce additional maintenance in the future. Therefore it makes sense to use a workflow that aims for completing step1 before step2 is approached. Thx for the fix. Approved now.

File: onlyoffice [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[Section]
```
Comment (julenuri) [inline onlyoffice:1]: File doesn't have .txt extension, so it's not added to RAPPS

File: onlyoffice [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+License = AGPL-3.0-only
+Description = OnlyOffice (formerly TeamLab), a powerful and free software office suite developed by Ascensio System SIA. Supports opening and saving documents in Microsoft Office formats.
+Category = 6
+URLSite = https://www.onlyoffice.com/
+URLDownload = https://download.onlyoffice.com/install/desktop/editors/windows/distrib/onlyoffice/DesktopEditors_x86.exe
+SHA1 = 3a9b1cea5e088b3462dc0f942e34b3a2373be5ee
+SizeBytes = 206759344
+
+[Section.0419]
+Description = OnlyOffice (в прошлом Teamlab) — мощный офисный пакет с открытым исходным кодом, разработанный компанией Ascensio System SIA. Поддерживает открытие и сохрание документов в форматах Microsoft Office.
```
Comment (julenuri) [inline onlyoffice:13]: No newline at end of file

File: onlyoffice (hunk 1 above)
Comment (JoachimHenze) [reply onlyoffice:1]: I fixed that with https://github.com/reactos/rapps-db/commit/238ebb848f89d1852985d19f0f51f79b2298825e
Thx for the mentioning @julenuri

**Comment by binarymaster** [comment]
I've fixed the download link to an XP compatible one in 671ea5cf405aea7101aed2f260e0e3eb3a9f964e.

There's also older version 6.1.0.94 [here](https://web.archive.org/web/20210214044140if_/http://download.onlyoffice.com/install/desktop/editors/windows/distrib/onlyoffice/DesktopEditors_x86_xp.exe) (207903672 bytes, SHA1 = `6e1e2aa707522bcbb98a589f426d00c4664f5944`), but I wasn't able to find version 7.3.0.184 which was initially added in this PR.

**Pull request merged**